README.md

```diff
@@ -96,12 +98,26 @@ python -m http.server PORT
 
 ## Furhat
 
-TBD
+The Furhat skill that facilitates the communication between the Rasa server and the Furhat robot is located [here](ui\furhat-skills). The skill should be configured by changing the IP to the Rasa server, compiled, and started. To compile the skill run:
```

Comment:
Nit: instead of `located [here]` you could say `located under [ui/furhat-skills]`. The link part should use a forward slash instead of a backslash.

Reply:
Done